src/demos/error/error-demo.component.html

```diff
@@ -112,9 +112,13 @@ <h3 class="sky-subsection-heading">
   Display an error message in a modal
 </h3>
 
+<p>
+  Do not use an image for errors when users can't perform an action such as when the panel and modal content doesn't load or display. For more information, see <a href="https://developer.blackbaud.com/skyux/design/guidelines/error-handling">error handling</a>.
```

Comment:
I think this first sentence might read slightly easier if we add a comma before `such as` to set off that clause and if we change `for` to `with` after `image`.

Reply:
Done